### PR TITLE
8280419: Remove dead code related to VerifyThread and verify_thread()

### PR DESCRIPTION
**Pull request by dholmes-ora**
This is code that was copied from the sparc port into the arm, ppc and s390 ports, but was never actually implemented.

I can't really test those platforms, but hopefully GHA builds will suffice, otherwise I need the various porters to confirm the changes.

Thanks.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8280419](https://bugs.openjdk.org/browse/JDK-8280419): Remove dead code related to VerifyThread and verify_thread()

### Reviewers
 * [Thomas Stuefe](https://openjdk.org/census#stuefe) (@tstuefe - **Reviewer**) ⚠️ Review applies to [96f6b52e](https://git.openjdk.org/jdk/pull/12506/files/96f6b52ea04abdfdc3dfa23d7341e02bd0016056)
 * [Lutz Schmidt](https://openjdk.org/census#lucy) (@RealLucy - **Reviewer**) ⚠️ Review applies to [96f6b52e](https://git.openjdk.org/jdk/pull/12506/files/96f6b52ea04abdfdc3dfa23d7341e02bd0016056)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12506/head:pull/12506` \
`$ git checkout pull/12506`

Update a local copy of the PR: \
`$ git checkout pull/12506` \
`$ git pull https://git.openjdk.org/jdk pull/12506/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12506`

View PR using the GUI difftool: \
`$ git pr show -t 12506`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12506.diff">https://git.openjdk.org/jdk/pull/12506.diff</a>

</details>

### CODE REVIEW

**Comment by dholmes-ora** [comment]
All cross-compilation builds successful.

**Comment by tstuefe** [comment]
Changes look good to me. GHAs are not there though.

I tested arm manually and it runs fine, rejecting VerifyThread as expected.

Maybe @RealLucy can eyeball the IBM platform changes.

**Review by tstuefe — APPROVED** [review]

File: src/hotspot/cpu/s390/interp_masm_s390.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 20223, Oracle and/or its affiliates. All rights reserved.
```
Comment (RealLucy) [inline src/hotspot/cpu/s390/interp_masm_s390.cpp:2]: Copyright year a bit far in the future

**Review by RealLucy — APPROVED** [review]
Changes for ppc and s390 look good to me.
There is one copyright typo you may want to fix.

**Comment by dholmes-ora** [comment]
> GHAs are not there though.

@tstuefe , sorry the GHA was a manual run:

https://github.com/dholmes-ora/jdk/actions/runs/4140195206

File: src/hotspot/cpu/s390/interp_masm_s390.cpp (hunk 1 above)
Comment (dholmes-ora) [reply src/hotspot/cpu/s390/interp_masm_s390.cpp]: Fixed

**Comment by dholmes-ora** [comment]
Thanks for the reviews @tstuefe  and @RealLucy !

**Pull request closed without merging**